src/test/components/FlameGraph.test.js

```diff
@@ -47,6 +47,7 @@ const GRAPH_HEIGHT = 300;
 
 describe('FlameGraph', function() {
   afterEach(removeRootOverlayElement);
+
```

Comment:
nitpick: Not a big fan of the `beforeEach` `afterEach` spacing rule, but it's fairly minor. I definitely prefer the consistent spacing on tests overall though.

Reply:
yeah I agree, I think this is configurable, I'll have a look

Reply:
I manually configured rules for just `test` and `describe`.